packages/core/src/lib/implementation/read-code-pushup-config.integration.test.ts

```diff
@@ -1,64 +0,0 @@
-import { dirname, join } from 'path';
-import { fileURLToPath } from 'url';
-import { describe, expect } from 'vitest';
-import { readCodePushupConfig } from './read-code-pushup-config';
-
-describe('readCodePushupConfig', () => {
```

Comment:
By replacing the integration tests with unit tests, we are losing test coverage when using actual file system. While it is okay to test most cases using mocked file system, at least a happy path should stay without mocks.